portfolio/src/main/webapp/style.css

```diff
@@ -1,3 +1,24 @@
+.card {
+  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+  transition: 0.3s;
+  width: 15%;
```

Comment:
I would recommend not using a percentage-based width here - they look pretty cramped on a smaller screen. I'd probably go with pixel-based, and possibly change depending on the screen size using a media query.